dummy/jwtAuth/auth.js

```diff
@@ -11,23 +11,24 @@ const verifyUser = (req, res, next) => {
 	const token = req.headers['x-access-token'];
 	if (!token) {
 		res.status(301).json({
-			success: false,
-			message: 'authentication failed, please login'
+			status: 301,
+			error: 'authentication failed, please login'
 		});
 	}
 
 	try {
 		const decoded = jwt.verify(token, secret);
 		if (!decoded) {
 			res.status(401).json({
-				message: 'authentication failed'
+			error: 'authentication failed'
 			});
 		}
 		req.decoded = decoded;
 		next();
 	} catch (error) {
 		res.status(400).json({
-			message: 'invalid token, you are not a valid user'
+			status:400,
+			error: 'invalid token, you are not a valid user'
```

Comment:
Consider changing this message to a more user-friendly response. You could say `Authentication failed, consider logging in again`.